File: src/android/AuthTokenCreationFactory.java
package edu.berkeley.eecs.emission.cordova.opcodeauth;

import android.content.Context;

import edu.berkeley.eecs.emission.cordova.connectionsettings.ConnectionSettings;

/**
 * Created by shankari on 8/19/17.
 *
 * Factory to generate AuthTokenCreator instances based on the connection settings
 */

public class AuthTokenCreationFactory {
    public static AuthTokenCreator getInstance(Context ctxt) {
        return new PromptedAuth(ctxt);
    }
}

File: src/ios/AuthTokenCreationFactory.m
//
//  AuthTokenCreationFactory.m
//  emission
//
//  Created by Kalyanaraman Shankari on 8/19/17.
//
//

#import "AuthTokenCreationFactory.h"
#import "PromptedAuth.h"

@implementation AuthTokenCreationFactory

+(id<AuthTokenCreator>) getInstance
{
    return [PromptedAuth sharedInstance];
}


@end
